lib/plugin.template.js

```diff
@@ -158,6 +158,7 @@ export default (ctx, inject) => {
     // Axios creates only one which is shared across SSR requests!
     // https://github.com/mzabriskie/axios/blob/master/lib/defaults.js
     headers: {
+      'Accept-Encoding': process.browser ? 'gzip, deflate, br' : 'gzip, deflate',
```

Comment:
This could be better to not change client-side defaults only set it in server side.

Reply:
PS `Accept-Encoding` is forbidden to set in Browsers: 

- https://github.com/axios/axios/issues/542#issuecomment-262146813
- https://xhr.spec.whatwg.org/#the-setrequestheader()-method
- https://stackoverflow.com/questions/22362169

Reply:
PS `Accept-Encoding` is forbidden to set in Browsers: 

- https://github.com/axios/axios/issues/542#issuecomment-262146813
- https://xhr.spec.whatwg.org/#the-setrequestheader()-method
- https://stackoverflow.com/questions/22362169

Reply:
Good catch! That's true. Setting it on client-side shouldn't have an effect at all

Reply:
updated.